lib/store_model/types/one_base.rb

```diff
@@ -57,8 +57,18 @@ def handle_unknown_attribute(value, exception)
         attribute = exception.attribute.to_sym
         value_symbolized = value.symbolize_keys
 
-        cast_value(value_symbolized.except(attribute)).tap do |configuration|
+        filtered_value = value_symbolized.except(attribute)
+
+        if value_symbolized.key?(:attributes)
+          filtered_value[:attributes] = filtered_value[:attributes].except(attribute)
```

Comment:
I guess we could make this check before `.exept`, i.e.

```ruby
value_symbolized = value_symbolized[:attributes] if value_symbolized.key?(:attributes)
filtered_value = value_symbolized.except(attribute)
```

Reply:
Cool, that makes sense to me (though I think we should be clear that mixing `attributes` and other top-level values isn't possible... will add a spec)